diff --git a/core/testcasecontroller/algorithm/paradigm/joint_inference/joint_inference.py b/core/testcasecontroller/algorithm/paradigm/joint_inference/joint_inference.py
index df5e46920..d57af2a1b 100644
--- a/core/testcasecontroller/algorithm/paradigm/joint_inference/joint_inference.py
+++ b/core/testcasecontroller/algorithm/paradigm/joint_inference/joint_inference.py
@@ -15,12 +15,14 @@
 """Cloud-Edge Joint Inference"""
 
 import os
+
 from tqdm import tqdm
 
-from core.common.log import LOGGER
 from core.common.constant import ParadigmType
+from core.common.log import LOGGER
 from core.testcasecontroller.algorithm.paradigm.base import ParadigmBase
 
+
 class JointInference(ParadigmBase):
     """
     Cloud-Edge-JointInference:
@@ -71,14 +73,8 @@ def set_config(self):
         os.makedirs(inference_output_dir, exist_ok=True)
 
         LOGGER.info("Loading dataset")
-        # self.inference_dataset = self.dataset.load_data(
-        #     self.dataset.test_data_info,
-        #     "inference"
-        # )
-        source = self.dataset.test_url if hasattr(self.dataset, 'test_url') \
-            else self.dataset.test_data_info
         self.inference_dataset = self.dataset.load_data(
-            source,
+            self.dataset.test_url,
             "inference"
         )
 
